packages/basic-crawler/src/internals/basic-crawler.ts

```diff
@@ -34,7 +34,7 @@ import {
     purgeDefaultStorages,
     validators,
 } from '@crawlee/core';
-import type { GotOptionsInit, OptionsOfTextResponseBody, Response as GotResponse } from 'got-scraping';
+import type { Method, OptionsInit, Response as GotResponse } from 'got-scraping';
```

Comment:
whats the difference? maybe it would be better to fix the `GotOptionsInit` interfce instead?

Reply:
The Got one is pure Got interface, the non-Got one is the proper (got-scraping) one

Reply:
oh right, i tought its the other way around 🤦